.github/workflows/ci.yml

```diff
@@ -135,6 +135,15 @@ jobs:
                 ARTIFACT_NAME: ${{ github.ref_name == 'nightly' && 'woocommerce-trunk-nightly.zip' || 'woocommerce.zip' }}
               working-directory: ${{ matrix.projectPath }}
               run: node ./tests/e2e-pw/bin/override-wp-env-plugins.js
+
+            - name: 'Cache Playwright Downloads'
+              id: 'cache-playwright-downloads'
+              if: ${{ matrix.testEnv.shouldCreate }}
+              uses: 'actions/cache@ab5e6d0c87105b4c9c2047343972218f562e4319'
+              with:
+                path: '~/.cache/ms-playwright/'
+                key: "${{ runner.os }}-playwright-${{ hashFiles( '**/pnpm-lock.yaml' ) }}"
+                restore-keys: '${{ runner.os }}-playwright-'
```

Comment:
Solid improvement here.  It looks like it takes it from >1 min to 20 sec.

Reply:
What is this caching exactly? Is it the browsers binaries?

Reply:
If it's the browser's binaries shouldn't the caching happen after the env setup step? That's where `playwright install` runs.

Reply:
Yes, it's chromium plus ffmpeg binaries. 

It's placed before as we try to restore the cache before Playwright downloading anything.
A cache update will be attempted in `Post Cache Playwright Downloads` step (automatic post run hook).